cmd/autofix_test.go

```diff
@@ -1,13 +1,49 @@
 package cmd
 
 import (
+	"bufio"
+	"bytes"
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-
-	"github.com/go-test/deep"
 )
 
+func compareFiles(file1, file2 string) bool {
+	f1, err := os.Open(file1)
+	if err != nil {
+		return false
+	}
+
+	f2, err := os.Open(file2)
+	if err != nil {
+		return false
+	}
+
+	s1 := bufio.NewScanner(f1)
+	s2 := bufio.NewScanner(f2)
+
+	for s1.Scan() {
+		s2.Scan()
+		if !bytes.Equal(s1.Bytes(), s2.Bytes()) {
+			return false
+		}
+	}
+	return true
+}
+
+func assertEqualWorkloads(assert *assert.Assertions, resource1, resource2 []Resource) {
+	file1 := "/tmp/dat1"
```

Comment:
Would it be beneficial to use io.ioutil.TempFile on the off chance that this file name is used by another program?